coordinator/src/main/java/org/apache/uniffle/coordinator/web/Response.java

```diff
@@ -22,18 +22,18 @@ public class Response<T> {
   private static final int ERROR_CODE = -1;
   private int code;
   private T data;
-  private String errMsg;
+  private String msg;
 
   public Response() {}
 
-  public Response(int code, T data, String errMsg) {
+  public Response(int code, T data, String msg) {
     this.code = code;
     this.data = data;
-    this.errMsg = errMsg;
+    this.msg = msg;
   }
 
   public static <T> Response<T> success(T data) {
-    return new Response<>(SUCCESS_CODE, data, null);
+    return new Response<>(SUCCESS_CODE, data, "success");
```

Comment:
Why returning `success`? What is the purpose?

Reply:
> Why returning `success`? What is the purpose?

This method is the success method, but the return message does not have any message, the front end can also judge the success of the message.